### ISSUE TITLE
Can't use bindings more than once in mysql query

### ISSUE THREAD

**Issue by tobias-lewald** [Budibase/budibase#4767]
**Describe the bug**
I've set up a mysql query, but when I try to use a binding multiple times, the binding is not being replaced. Is there another way to achieve that or is it a bug?

**To Reproduce**
1. Connect MySQL Database
2. Add new Query
3. Add new Binding: name -> test, default -> test
4. Change query to: `SELECT '{{ test }}' as t1;`
5. Run query -> everything works fine
6. Change query to: `SELECT '{{ test }}' as t1, '{{ test }}' as t2;`
7. Run query -> now the bindings are not being replaced

**Expected behavior**
Now both handlebars expressions should be replaced.

**Desktop:**
 - OS: macOS Monterey
 - Browser Firefox
 - Version 97.0.1

**Comment by kovrom** [comment]
Posted the same issue in the Discussions almost at the same time as you :) Will repost it here, just in case too
Self hosted.
Updated from 1.0.26 to 1.0.73. Every PostgreSQL query where one binding was used multiple times stopped working. Work around for now is to create multiple bindings with the same values. But it looks like a bug and definitely not very productive. Any info on that? Thank you.
For example following will not work anymore. Let's say `binding = 0`:

```
 SELECT something AS ValueOne, 
(SELECT something2 FROM table WHERE column ='{{binding}}') AS ValueTwo
FROM table
WHERE column2 ='{{binding}}';
```

To make it work I have to create 2 bindings: `binding=0 `and `binding1=0`. And do:

```
 SELECT something AS ValueOne, 
(SELECT something2 FROM table WHERE column ='{{binding}}') AS ValueTwo
FROM table
WHERE column2 ='{{binding1}}';
```

**Comment by mike12345567** [comment]
This issue will be resolved in our upcoming 1.0.80 release, which should be released in the next day or two.
